### DISCUSSION TITLE
How to rotate marker using phone gps?

### DISCUSSION THREAD

**Discussion by ShourovRoy** [fleaflet/flutter_map#1410] [Q&A] [answered]
Flutter map is one of the best thing I have even seen but I have a question. How can we rotate a Marker according to phone rotation using gps. I am confused . Can anyone help me please?

**Comment by JaffaKetchup** [comment, marked as answer]
Hi @ShourovRoy,
Yes this is possible. You'll need to DIY though, or use the plugin 'flutter_map_location_market' if that's what you're trying to do.
If you want to do it manually, have a look at that project for inspiration anyway.
